webui/react/src/pages/ExperimentList.tsx

```diff
@@ -7,21 +8,39 @@ import { makeClickHandler } from 'components/Link';
 import linkCss from 'components/Link.module.scss';
 import Page from 'components/Page';
 import StateSelectFilter from 'components/StateSelectFilter';
+import TableBatch from 'components/TableBatch';
 import Toggle from 'components/Toggle';
 import UserSelectFilter from 'components/UserSelectFilter';
 import Auth from 'contexts/Auth';
 import Users from 'contexts/Users';
+import handleError, { ErrorLevel, ErrorType } from 'ErrorHandler';
 import usePolling from 'hooks/usePolling';
 import { useRestApiSimple } from 'hooks/useRestApi';
 import useStorage from 'hooks/useStorage';
-import { getExperimentSummaries } from 'services/api';
+import { setupUrlForDev } from 'routes';
+import {
+  archiveExperiment, getExperimentSummaries, killExperiment, launchTensorboard, setExperimentState,
+} from 'services/api';
 import { ExperimentsParams } from 'services/types';
-import { ALL_VALUE, Experiment, ExperimentFilters, ExperimentItem } from 'types';
+import {
+  ALL_VALUE, Command, Experiment, ExperimentFilters, ExperimentItem, RunState, TBSourceType,
+} from 'types';
+import { openBlank } from 'utils/routes';
 import { filterExperiments, processExperiments } from 'utils/task';
+import { cancellableRunStates, isTaskKillable, terminalRunStates, waitPageUrl } from 'utils/types';
 
 import css from './ExperimentList.module.scss';
 import { columns } from './ExperimentList.table';
 
+enum Action {
+  Activate = 'Activate',
+  Archive = 'Archive',
```

Comment:
suggestion: if unarchive is just another entry in the list might be a good addition to counter batch archive. (missing from elm)